jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryClient.java

```diff
@@ -64,7 +64,7 @@
     private final RegistryEndpointRequestProperties registryEndpointRequestProperties;
     private final FailoverHttpClient httpClient;
 
-    private String userAgent = "";
+    @Nullable private String userAgent = "";
```

Comment:
This should not assign `""`. The default should be null.

Reply:
Makes sense.